tests/data/greek-verb-paradigm.test.js

```diff
@@ -0,0 +1,856 @@
+  })
+
+  it('29 - checked Verb Paradigm 29 - τιθέσθων', async () => {
+    const inflectionsViewSet = await getInflectionSet('τιθέσθων', Constants.LANG_GREEK)
+    // console.info(inflectionsViewSet.matchingViews)
+    
+    expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
+    expect(inflectionsViewSet.matchingViews.length).toEqual(1)
+
+    expect(inflectionsViewSet.matchingViews[0].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[0].title).toEqual('τίθημι: Present System Middle-Passive')
+    expect(inflectionsViewSet.matchingViews[0].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[0].paradigm.paradigmID).toEqual('verbpdgm29')
+    expect(inflectionsViewSet.matchingViews[0].paradigm.title).toEqual('τίθημι: Present System Middle-Passive')
+
+    expect(inflectionsViewSet.matchingViews[0].linkedViews.length).toEqual(0)
+  })
+
+  it('30 - checked Verb Paradigm 30 - ἔθεσαν', async () => {
+    const inflectionsViewSet = await getInflectionSet('ἔθεσαν', Constants.LANG_GREEK)
+    // console.info(inflectionsViewSet.matchingViews)
+    
+    expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
+    expect(inflectionsViewSet.matchingViews.length).toEqual(1)
+
+    expect(inflectionsViewSet.matchingViews[0].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[0].title).toEqual('τίθημι: Aorist System Active')
+    expect(inflectionsViewSet.matchingViews[0].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[0].paradigm.paradigmID).toEqual('verbpdgm30')
+    expect(inflectionsViewSet.matchingViews[0].paradigm.title).toEqual('τίθημι: Aorist System Active')
+
+    expect(inflectionsViewSet.matchingViews[0].linkedViews.length).toEqual(0)
+  })
+
+  it('31 - checked Verb Paradigm 31 - ἐθέμεθα', async () => {
+    const inflectionsViewSet = await getInflectionSet('ἐθέμεθα', Constants.LANG_GREEK)
+    // console.info(inflectionsViewSet.matchingViews)
+    
+    expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
+    expect(inflectionsViewSet.matchingViews.length).toEqual(1)
+
+    expect(inflectionsViewSet.matchingViews[0].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[0].title).toEqual('τίθημι: Aorist System Middle')
+    expect(inflectionsViewSet.matchingViews[0].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[0].paradigm.paradigmID).toEqual('verbpdgm31')
+    expect(inflectionsViewSet.matchingViews[0].paradigm.title).toEqual('τίθημι: Aorist System Middle')
+
+    expect(inflectionsViewSet.matchingViews[0].linkedViews.length).toEqual(0)
+  })
+
+  it('32 - no matches - checked Verb Paradigm 32', async () => {
+  })
+
+  it('33 - no matches - checked Verb Paradigm 33', async () => {
+  })
+
+  it('34 - no matches - checked Verb Paradigm 34', async () => {
+  })
+
+  it('35 - no matches - checked Verb Paradigm 35', async () => {
+  })

```

Comment:
these tables are not working?

Reply:
the same as previous

Reply:
@monzug have you found any words that (correctly) produce paradigms 32-35?

Reply:
yes. for  verbpdgm32     ἵην      or        ἱέτω    or   ἵημι       ---  stemtype is irreg_mi
when stemtype is ew_pr,  it goes to other tables,  such as  for  ἱῇ

Reply:
same for table 33.

Reply:
for verbpdgm34,  stemtype for   ἀφῆκα     is   aor1  so it opens the verbpdgm8 if active or verbpdgm9 if middle.
so for verbpadgm34 and 35, the answer is no.

Reply:
Thank you, @monzug !
I have added tests for verbpdgm32 and verbpdgm33